Vagrantfile

```
@@ -0,0 +1,53 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000, host_ip: "0.0.0.0"
+	config.vm.provision "shell", privileged: false, inline: <<-SHELL
+		# echo "Provisioning virtual machine..."
+		sudo apt-get update -y
+		
+		# echo "Installing dependencies and compiler"
+		sudo apt-get install -y make build-essential libssl-dev zlib1g-dev \
+		libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+		libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+
+		# echo "Installing pyenv";
+		git clone https://github.com/pyenv/pyenv.git $HOME/.pyenv
+		echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile;
+		echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile;
+		echo 'eval "$(pyenv init --path)"' >> ~/.profile;
+		source ~/.profile;
+	  
+		# echo "Installing python";
+		pyenv install 3.7.10;
+		
+		#echo "Installing poetry"
+		curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+	SHELL
+	config.trigger.after :up do |trigger|
+		 trigger.name = "Launching App"
+		 trigger.info = "Running the TODO app setup script"
+		 trigger.run_remote = {privileged: false, inline: " 
+		 	sudo apt-get update			 
+			sudo pip install Flask
+			sudo pip install python-dotenv
+			sudo apt-get install -yq \
+			  python3-venv \
+			  python-dev \
+			  python-pip \
+			  python-virtualenv \
+			  libpq-dev
+			cd /vagrant/todo_app
+		
+			flask run --host='0.0.0.0'
```

Why are you using pip instead of poetry?
Why install python via apt-get when you already went through all the effort of installing python 3.7.10 via pyenv in your provisioning script?

```suggestion
		cd /vagrant
		poetry install
		poetry run flask run --host='0.0.0.0'
```

`/vagrant` is the folder that will be synced with your project so you should run commands from there like you run them from this project directory.